src/dual_sets.jl

```diff
@@ -0,0 +1,16 @@
+# This should be putted in MOIU and used for Dualization later
+function dual_function_in_set(F::MOI.AbstractFunction, S::MOI.AbstractSet)
+    return _dual_set(F, S)
+end
+
+function _dual_set(F::SAF{T}, S::MOI.GreaterThan) where T
+    return MOI.GreaterThan(zero(T))
+end
+
+function _dual_set(F::SAF{T}, S::MOI.LessThan{T}) where T
+    return MOI.LessThan(zero(T))
+end
+
+function _dual_set(F::SAF{T}, S::MOI.EqualTo) where T
+    return # Reals ?
```

Comment:
reals is dual to equal, yes. not sure if it makes sense to add a reals constraint (which would just be removed by solver wrapper).

Reply:
I think it is better to return `nothing` and just don't add a constraint in `Reals`